ccf/tests/test_models.py

```diff
@@ -3,13 +3,25 @@
 from app.models import Job
 
 
+@pytest.fixture
```

Comment:
I would prefer this be in a test_utils.py file that way we can create users on command in any tests we may need to in the future.

Reply:
Addressed in latest commit.